app/components/bali/navbar/burger/component.rb

```diff
@@ -21,6 +24,23 @@ def call
                       ])
           end
         end
+
+        private
+
+        def configure_attrs
+          attrs = type == :main ? attrs_for_main : attrs_for_alt
+
+          prepend_action(options, attrs[:action])
+          prepend_data_attribute(options, 'navbar-target', attrs[:target])
+        end
+
+        def attrs_for_main
+          @attrs_for_main ||= { target: 'burger', action: 'navbar#toggleMenu' }
+        end
+
+        def attrs_for_alt
+          @attrs_for_alt ||= { target: 'altBurger', action: 'navbar#toggleAltMenu' }
+        end
```

Comment:
no veo necesario memoizarlos, porque solo lo estas llamando 1 sola vez y además es un hash estático.

```suggestion
        def attrs_for_main
          @attrs_for_main ||= { target: 'burger', action: 'navbar#toggleMenu' }
        end

        def attrs_for_alt
          @attrs_for_alt ||= { target: 'altBurger', action: 'navbar#toggleAltMenu' }
        end
```